lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1243,6 +1242,40 @@ def get_service_name_from_init_system(self, service):
             else:
                 return service
 
+    def get_magma_services(self) -> List[str]:
```

Comment:
To be honest I liked the KISS list more. Now I always have to think about docker and systemd when determining which service are restarted and go through a yaml file and then understand this code.

Reply:
The differentiation between docker and systemd has to happen at some point (as well as the magma@ prefix handling). Duplicating the entries of the yaml file contradicts the intention to have as much as possible only in one place. Adding / changing one service you must know that there is a second list somewhere.